### PR TITLE
Sloppy language

### PR DESCRIPTION
**Pull request by dianawi**

### CODE REVIEW

File: system/languages/en.php [hunk 1]
```diff
@@ -257,6 +260,7 @@
 		"ml-successfully-unsubscribed" => "you have successfully unsubscribed",
 		"ml-successfully-created" => "successfully created",
 		"ml-successfully-updated" => "successfully updated",
+		"ml-comment-please-confirm" => "Thank you for your comment. Please confirm your identity by clicking the confirmation links that we have sent you by email.",
```
Comment (matthijskooijman) [inline system/languages/en.php]: I can't find any reference to this string in the code?

Comment (matthijskooijman) [reply system/languages/en.php]: Ah, seems this was added in #227 (both the string definition and the usage). I think this should be dropped from this commit.

Comment (matthijskooijman) [reply system/languages/en.php]: Scratch that, it seems #227 only adds the strings, but does not use them anywhere...

Comment (tammoterhark) [reply system/languages/en.php]: Looks to me like `ml` refers to mailinglists. Can they have comments?

**Comment by matthijskooijman** [comment]
Looks good at first glance. I think this can be merged after testing.

**Comment by matthijskooijman** [comment]
Tested, works. I rebased on top of master, merging now.

**Pull request merged**